package.json

```diff
@@ -1,69 +1,48 @@
 {
   "name": "ngx-flip",
-  "version": "0.0.1",
-  "license": "MIT",
-  "author": "Ritesh Srivastava",
-  "description": "Simple Angualr based plug and play Flip component",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ritsrivastava01/ngxFlip.git"
-  },
-  "homepage": "https://github.com/ritsrivastava01/ngxFlip",
-  "bugs": {
-    "url": "https://github.com/ritsrivastava01/ngxFlip"
-  },
-  "keywords": [
-    "angular",
-    "angular2",
-    "angular 2",
-    "angular4",
-    "angular Flip component",
-    "angular 2 Flip"
-  ],
+  "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e",
-    "packagr": "ng-packagr -p ng-package.json"
+    "e2e": "ng e2e"
   },
-  "private": false,
+  "private": true,
```

Comment:
Hi @tonymullen ,
is there any reason to do that?

Reply:
Oops. No. I overlooked that. Fixed now.